modules/mobile_session_impl.lua

```diff
@@ -185,7 +186,8 @@ end
 -- @tparam table ignoreHeartBeatAck Access table for ignore heartbeat ACK from SDL flag
 -- @tparam table regAppParams Mobile application parameters
 -- @treturn MobileSessionImpl Constructed instance
-function MSI.MobileSessionImpl(session_id, correlation_id, test, connection, sendHeartbeatToSDL, answerHeartbeatFromSDL, ignoreHeartBeatAck, regAppParams)
+function MSI.MobileSessionImpl(session_id, correlation_id, test, connection, activateHeartbeat,
```

Comment:
@aderiabin do not forget to update description

Reply:
@LuxoftAKutsan Done (see f6a63fb5e7166ecbb3573baa29347f4a5a3d1c28)